cli/src/utils/compareSchema.ts

```diff
@@ -17,8 +21,12 @@ export function compareSchemas(
 
   // Compare columns
   for (const table of sourceTables) {
-    const sourceColumns = Object.keys(source.tables?.[table]?.columns ?? {});
-    const targetColumns = Object.keys(target.tables?.[table]?.columns ?? {});
+    const sourceColumns = Object.keys(source.tables?.[table]?.columns ?? {}).filter(
```

Comment:
Since we use this function with `adaptTable` = true, adapt will add the internal columns to the migration if they don't exist causing it to fail.